src/sentry/migrations/0584_backfill_alert_rule_user_and_team.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.28 on 2023-04-26 19:54
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_alert_rule_user_and_team(apps, schema_editor):
+    AlertRule = apps.get_model("sentry", "AlertRule")
+    for ar in RangeQuerySetWrapperWithProgressBar(
+        AlertRule.objects.filter(owner_id__isnull=False).select_related("owner")
```

Comment:
Just a general warning/reminder: When we use `RangeQuerySetWrapperWithProgressBar`, it adds an order by id to the queryset. So unless you have an index on `owner_id, id` it can make each batch fetch slow.

This table only has 100k rows, so it's fine, but just raising a bit of awareness.

Reply:
Good call, thank you!

Reply:
To be honest, I can remove owner_id filter here and just use a conditional.